multi-agent/lib/mas/prompts/service.py

```
@@ -0,0 +1,411 @@
+
+        if self._schedule_port:
+            try:
+                temporal_id = self._schedule_port.create_schedule(prompt)
+            except Exception:
+                self._repo.delete(prompt.id)
+                raise
+            prompt = prompt.model_copy(update={"temporal_schedule_id": temporal_id})
+            if not self._repo.update(prompt):
+                try:
+                    self._schedule_port.delete(temporal_id)
+                except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)